applications/models/application.py

```diff
@@ -190,7 +189,7 @@ def _(self) -> ApplicationStatusChoice:
     def all_sections_allocated() -> bool:
         from applications.models import ApplicationSection
 
-        exists = ~models.Exists(
+        return ~models.Exists(  # type: ignore[return-value]
```

Comment:
These lookup properties were purposely done this way (assign to a variable, return that variable), since using the `# type: ignore[...]` will ignore all typing issues in the expression itself, which makes it susceptible to other typing issues. I think it's fine to fix them like this, but I might change them back later.

Reply:
Good to know! That's perfectly fine, just need to add ignore for automatic formatting too :)